## References

Discussions

* https://www.biostars.org/p/1332/
* https://www.biostars.org/p/91821/

## Genome Browsers

* [UCSC Genome Browser](https://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=424562525_yNMUAd14eoZBR0qwjQZ8xVVixHi2)
* Description of [snp142 table](http://ucscbrowser.genap.ca/cgi-bin/hgTables?hgsid=1186698_UHMlYABco6ZL3qYUtaaB41BV859c&hgta_doSchemaDb=hg19&hgta_doSchemaTable=snp142) (searched by googling)
* [List of data bases](http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/)
* RMySQL [R code](http://www.gettinggeneticsdone.com/2011/12/query-mysql-database-from-r-using.html) to access UCSC data base

### Tools

1) [bioruby-ucsc-api](https://github.com/misshie/bioruby-ucsc-api)

* [Presentation](http://www.slideshare.net/misshie/the-ruby-ucsc-api-ismb2012)
* [An example](https://www.biostars.org/p/8920/#8928)
* A tutorial http://hackersome.com/p/Diego-F-Pereira/bioruby-ucsc-api
* MySQL connection server proble
      * http://redmine.soe.ucsc.edu/forum/index.php?t=msg&goto=13723&S=85d8e65f80fc3cbbd62d680143c21982

2) Python bindings [pyucsc](https://github.com/PopulationGenetics/pyucsc)

3) Plots

* http://genome.sph.umich.edu/wiki/LocusZoom_Standalone

#### R packages

* http://www.bioconductor.org/packages/release/bioc/html/biomaRt.html

Plots

* mapsnp: https://github.com/csuzfq/mapsnp_pkg
     * Paper with plots: http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0123609#sec011
* GenomeGraphs: http://bioconductor.org/packages/2.3/bioc/html/GenomeGraphs.html

## Annotation tools

* SnpEff http://snpeff.sourceforge.net/index.html
    * SnpEff supports more db than Anovar
* Anovar http://annovar.openbioinformatics.org/en/latest/

## 1K genomic maps

* https://github.com/joepickrell/1000-genomes-genetic-maps

## List of genes

* https://www.biostars.org/p/81738/#81745

## Gusev et al, 2014

* Variante type https://data.broadinstitute.org/alkesgroup/ANNOTATIONS/ via http://www.hsph.harvard.edu/alkes-price/software/
